src/components/PeoplePage.tsx

```diff
@@ -1,12 +1,138 @@
-import { PeopleFilters } from './PeopleFilters';
+import { useEffect, useMemo, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { Person } from '../types';
+import { getPeople } from '../api';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { PeopleFilters } from './PeopleFilters';
+
+enum SortType {
+  NAME = 'Name',
+  SEX = 'Sex',
+  BORN = 'Born',
+  DIED = 'Died',
```

Comment:
the enum's elements should be in CamelCase
```suggestion
  Name = 'Name',
  Sex = 'Sex',
  Born = 'Born',
  Died = 'Died',
```